nft-minter/src/common_storage.rs

```diff
@@ -23,6 +23,7 @@ pub struct BrandInfo<M: ManagedTypeApi> {
 #[derive(TypeAbi, TopEncode, TopDecode, NestedEncode, NestedDecode, PartialEq, Debug)]
 pub struct MintPrice<M: ManagedTypeApi> {
     pub start_timestamp: u64,
+    pub opt_end_timestamp: Option<u64>,
```

Comment:
Why is this in a `MintPrice` struct? Should we refactor/rename it?

Reply:
Moved the timestamps to BrandInfo.